sqlstore/src/main/resources/db/migration/V8813__Dilution_CreatedDate.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `LibraryDilution` ADD COLUMN `created` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
```

Comment:
This will fill the table with the time the upgrade is run, so can you also do:

```
UPDATE LibraryDiltuion SET created = creationDate;
```
to backfill this historical data?